sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/UserPrincipalManager.java

```diff
@@ -181,31 +185,56 @@ private static boolean isAADIssuer(String issuer) {
             || issuer.startsWith(STS_CHINA_CLOUD_API_ISSUER);
     }
 
+    private boolean isAADTenant(String tenant) {
```

Comment:
This function name is confusing, I suppose all tenants are AAD tenant right?

Reply:
yes,I'm going to modify it